_posts/2020-03-11-sr2020-covid-19-updates.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: news
+title: COVID-19 Coronavirus Student Robotics Updates
+---
+
+As you will be aware, the COVID-19 situation is quickly evolving, having recently been delcared a pandemic. 
+Many large events are being cancelled and we want to be public about how we are responding to the latest news.
+
+The Competition Team are working closely with the Student Robotics trustees so that we can make the most sensible decision.
+
+## The Competition
+
+At the time of writing, we are carefully considering whether the competition can go ahead. As it stands, we still intend to have the competition on the 18th and 19th April 2020.
```

Comment:
Having this be 2 sentences which start similarly reads slightly weirdly

```suggestion
At the time of writing, we still intend to have the competition on the 18th and 19th April 2020, however are actively and carefully considering whether this needs to change.
```

(Not entirely convinced by the second part of my suggestion)